ccm_web/client/src/pages/Home.tsx

```diff
@@ -24,15 +25,17 @@ const mergeSectionProps: FeatureCardProps = {
   description: 'Combine sections into one Canvas site for easier management',
   icon: <MergeTypeIcon fontSize='large' />,
   ordinality: 1,
-  route: '/merge-sections'
+  route: '/merge-sections',
+  component: MergeSections
```

Comment:
I think we're starting to get to the point where it does make sense to extract these objects from `Home`. They could go in a `features.ts` module that gets imported by `Home` and `App`. That would also give us an opportunity to simplify some code variable names from `featureCardProps` to `feature`. If it were me, I'd create a `Feature` interface that's identical to the `FeatureCardProps` one and then adjust the `FeatureCard` component to accept a `Feature`. I think it will help us out in the long run if we start to think of these objects in a way less attached to one use case (`*Card`).

Reply:
I refactored this, however the configuration still requires some UX specific configuration.
Now there is  feature.ts that contains non UX related config (everything except the icon and the component for the feature) and the rest is configured in FeatureCard.  That could be put into yet another ts file too, though its not clear to me that would  present any big advantage?